ftp/src/main/java/ch/cyberduck/core/ftp/FTPProtocol.java

```diff
@@ -36,6 +36,11 @@ public String getPrefix() {
         return String.format("%s.%s", FTPProtocol.class.getPackage().getName(), StringUtils.upperCase(this.getType().name()));
     }
 
+    @Override
+    public DirectoryTimestamp getDirectoryTimestamp() {
+        return DirectoryTimestamp.implicit;
```

Comment:
is it though?
I could see Microsoft FTP not adhering to this, or at least Microsoft FTP on NTFS drives, where modification dates are not propagated up the tree for directories.

Reply:
Yes, this also applies to UNIX. Modification dates are only updated for the parent folder and this assumption is not correct.

Reply:
Currently only Nextcloud/ownCloud have ETag and timestamps [propagate to the root](https://github.com/owncloud/client/wiki/Etags-and-file-ids#etag-change). Other WebDAV implementations *may* also support this.

Reply:
SFTP on Windows won't be implicit as well.

Reply:
So Windows would be direct parent folder only, not propagating up.

Reply:
Clarified in afff5eba that the definition only applies to direct parent folder.